docs/tutorials/cloud-agent.md

```diff
@@ -0,0 +1,160 @@
+
+## Create a deployment
+
+Following the example of the [Flow deployments](/tutorials/deployments/) tutorial, let's create a deployment that can be executed by the agent on this container instance. You'll also need a storage block configured to save your flow code as described in the [Storage and Infrastructure](/tutorials/storage/) tutorial.
+
+In an environment where you have [installed Prefect](/getting-started/installation/), create a new folder called `health_test`, and within it create a new file called `health_flow.py` containing the following code.
+
+
+```python
+import prefect
+from prefect import task, flow
+from prefect import get_run_logger
+
+
+@task
+def say_hi():
+    logger = get_run_logger()
+    logger.info("Hello from the Health Check Flow! 👋")
+
+
+@task
+def log_platform_info():
+    import platform
+    import sys
+    from prefect.server.api.server import SERVER_API_VERSION
+
+    logger = get_run_logger()
+    logger.info("Host's network name = %s", platform.node())
+    logger.info("Python version = %s", platform.python_version())
+    logger.info("Platform information (instance type) = %s ", platform.platform())
+    logger.info("OS/Arch = %s/%s", sys.platform, platform.machine())
+    logger.info("Prefect Version = %s 🚀", prefect.__version__)
+    logger.info("Prefect API Version = %s", SERVER_API_VERSION)
+
+
+@flow(name="Health Check Flow")
+def health_check_flow():
+    hi = say_hi()
+    log_platform_info(wait_for=[hi])
+```
+
+Now create a deployment for this flow script, making sure that it's configured to use the `test` queue on the `default-agent-pool` work pool.
+
+<div class='terminal'>
+```bash
+prefect deployment build --infra process --storage-block azure/flowsville/health_test --name health-test --pool default-agent-pool --work-queue test --apply health_flow.py:health_check_flow
+```
+</div>
+
+Once created, any flow runs for this deployment will be picked up by the agent running on this container instance.
+
+!!! note "Infrastructure and storage"
+    This Prefect deployment example was built using the [`Process`](/concepts/infrastructure/#process) infrastructure type and Azure Blob Storage. 
+
+    You might wonder why your deployment needs process infrastructure rather than [`DockerContainer`](/concepts/infrastructure/#dockercontainer) infrastructure when you are deploying a Docker image to ACI.
+
+    A Prefect deployment’s infrastructure type describes how you want Prefect agents to run flows for the deployment. With `DockerContainer` infrastructure, the agent will try to use Docker to spin up a new container for each flow run. Since you’ll be starting your own container on ACI, you don’t need Prefect to do it for you. Using a process deployment tells Prefect you want to agent to run flows by starting a process in your ACI container.
```

Comment:
```suggestion
    A Prefect deployment’s infrastructure type describes how you want Prefect agents to run flows for the deployment. With `DockerContainer` infrastructure, the agent will try to use Docker to spin up a new container for each flow run. Since you’ll be starting your own container on ACI, you don’t need Prefect to do it for you. Specifying process infrastructure on the deployment tells Prefect you want to agent to run flows by starting a process in your ACI container.
```